contracts/farm/LMV1toLMV2Migrator.sol

```diff
@@ -0,0 +1,159 @@
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+import "../openzeppelin/ERC20.sol";
+import "../openzeppelin/SafeERC20.sol";
+import "../openzeppelin/SafeMath.sol";
+import "../utils/AdminRole.sol";
+import "./ILiquidityMiningV1.sol";
+import "./ILiquidityMiningV2.sol";
+
+contract LMV1toLMV2Migrator is AdminRole {
+	using SafeMath for uint256;
+	using SafeERC20 for IERC20;
+	enum MigrationStates {
+		None,
+		FinishedPoolsMigration, //pools has been migrated to LiquidityMiningV2 (set by migratior contract)
+		FinishedUsersMigration, //users has been migrated to LiquidityMiningV2 (set by admin)
+		FinishedFundsMigration //funds has been migrated to LiquidityMiningV2 (set by migrator contract)
+	}
+
+	//represents de migration state from LiquidityMiningV1 to LiquidityMiningV2
+	MigrationStates public migrationState;
+
+	//LiquidityMiningV1 contract address
+	ILiquidityMiningV1 public liquidityMiningV1;
+
+	//LiquidityMiningV2 contract address
+	ILiquidityMiningV2 public liquidityMiningV2;
+
+	/// @dev it is true if the user has been already migrated
+	mapping(address => bool) public userMigrated;
+
+	/// @dev The SOV token
+	IERC20 public SOV;
+
+	event UserMigrated(address indexed user);
+
+	/* Modifiers */
+	modifier onlyPoolsMigrationState() {
+		require(migrationState == MigrationStates.None, "pools have already been migrated");
+		_;
+	}
+
+	modifier onlyUsersMigrationState() {
+		require(migrationState >= MigrationStates.FinishedPoolsMigration, "have to migrate pools first");
+		require(migrationState < MigrationStates.FinishedUsersMigration, "users have already been migrated");
+		_;
+	}
+
+	modifier onlyFundsMigrationState() {
+		require(migrationState >= MigrationStates.FinishedUsersMigration, "have to migrate users first");
+		require(migrationState < MigrationStates.FinishedFundsMigration, "funds have already been migrated");
```

Comment:
Same, why not `require(migrationState == MigrationStates.FinishedUsersMigration)` ?